src/main/webapp/WEB-INF/views/creationForm.jsp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Developer Form</title>
+</head>
+
+<body>
+
+  <center>
+
+    <h1>
+      <form action="/DeveloperApi/form" method="POST">
+
+        <input type="text" name="firstname" value="First Name..." onclick="this.value=''"/><br/>
```

Comment:
You use value + onclick to add some description to the field.
Bad design. Make the research and refactor the code.

Reply:
fixed